### PR TITLE
Verification Docs

### PR DESCRIPTION
**Pull request by TristanBarlow**
Added technical documentation for the verification platform signup fields

### CODE REVIEW

File: Verificaiton.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# License Verification Platform Technical Documentation
+The purpose of this document is to provide documentation for concepts relating to the [License Verification Platform](https://verificaiton.clicknclear.com) that do not fall under the API documentation.
```
Comment (jackbinney) [inline Verificaiton.md:2]: Broken link due to typo

File: Verificaiton.md
Comment (jackbinney) [inline Verificaiton.md:1]: File typo: `Verificaiton` -> `Verification`

File: Verificaiton.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Signup Fields and Signup Keys
+When setting up a season or event you may want to collect custom data during signup.
+
+To collect this data we recommend using Signup fields, which can be found on the season edit / event edit pages.
+When setting up a Signup field, you will notice the input 'Field Key'.
+
+Field keys can be used to identify certain fields between any season/event that you may be running.
+
+Our API endpoints will return the entries along with any signup field answers along with its associated field key.
```
Comment (jackbinney) [inline Verificaiton.md:12]: NIT: double usage of "along" may read better as:

> Our API endpoints will return the entries along with any signup field answers and the associated field keys (if set).

Etc.

File: Verificaiton.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+```
+https://verification.clicknclear.com/my-events/event/1/invite?token=e175e0c9e5d73fe2c2574c83b1e1d5dd&uniqueID=12345678
+```
+If you want to auto-fill the default email and name fields you can do that by providing the entryName & entryEmail query parameters.
+For example:
+```
+https://verification.clicknclear.com/my-events/event/1/invite?token=e175e0c9e5d73fe2c2574c83b1e1d5dd&uniqueID=12345678&entryName=Tristan&entryEmail=tristan@clicknclear.com
+```
+### Limitations
+Depending on the values you use to for auto-filling you may need to encode your values to be URI compatible.
```
Comment (jackbinney) [inline Verificaiton.md:30]: Typo: "you use to for" -> "you use for"

File: Verificaiton.md (hunk 1 above)
Comment (TristanBarlow) [reply Verificaiton.md]: ba64c5a

File: Verificaiton.md
Comment (TristanBarlow) [reply Verificaiton.md:1]: ba64c5a

File: Verificaiton.md (hunk 2 above)
Comment (TristanBarlow) [reply Verificaiton.md]: ba64c5a

File: Verificaiton.md (hunk 3 above)
Comment (TristanBarlow) [reply Verificaiton.md]: 433d19c

File: Verificaiton.md (hunk 1 above)
Comment (TristanBarlow) [reply Verificaiton.md]: Ahhh sorry mis-read: c27a558ce28c886be58ca9143d38b389bd44d264

**Review by jackbinney — APPROVED** [review]

**Pull request merged**